sceptre/cli/create.py

```diff
@@ -10,9 +10,15 @@
 @click.argument("path")
 @click.argument("change-set-name", required=False)
 @click.option("-y", "--yes", is_flag=True, help="Assume yes to all questions.")
+@click.option(
+    "-dr",
+    "--disable-rollback",
+    is_flag=True,
+    help="Disable the auto rollback and keep resources successfully created or updated",
+)
 @click.pass_context
 @catch_exceptions
-def create_command(ctx, path, change_set_name, yes):
+def create_command(ctx, path, change_set_name, yes, disable_rollback: bool):
```

Comment:
If we adopt the enable/disable suggestion above, we'll need to update this function signature:

```suggestion
def create_command(ctx, path, change_set_name, yes, disable_rollback: Optional[bool]):
```